app/http.php

```diff
@@ -0,0 +1,330 @@
+use Swoole\Table;
+use Swoole\Timer;
+use Utopia\App;
+use Utopia\Logger\Log;
+use Utopia\Logger\Logger;
+use Utopia\Logger\Adapter\AppSignal;
+use Utopia\Logger\Adapter\LogOwl;
+use Utopia\Logger\Adapter\Raygun;
+use Utopia\Logger\Adapter\Sentry;
+use Utopia\Balancing\Algorithm;
+use Utopia\Balancing\Algorithm\Random;
+use Utopia\Balancing\Algorithm\RoundRobin;
+use Utopia\Balancing\Balancing;
+use Utopia\Balancing\Option;
+use Utopia\CLI\Console;
+use Utopia\Registry\Registry;
+use Utopia\Swoole\Request;
+use Utopia\Swoole\Response;
+
+Runtime::enableCoroutine(true, SWOOLE_HOOK_ALL);
+
+App::setMode((string) App::getEnv('OPEN_RUNTIMES_PROXY_ENV', App::MODE_TYPE_PRODUCTION));
+
+// Setup Registry
+$register = new Registry();
+
+$register->set('state', function () {
+    $count = \count(\explode(',', (string) App::getEnv('OPEN_RUNTIMES_PROXY_EXECUTORS', '')));
+    $state = new Table($count);
+    $state->column('hostname', Swoole\Table::TYPE_STRING, 128); // Same as key of row
+    $state->column('status', Swoole\Table::TYPE_STRING, 8); // 'online' or 'offline'
+    $state->column('state', Swoole\Table::TYPE_STRING, 16384); // State as JSON
+    $state->create();
+    return $state;
+});
+
+$register->set('logger', function () {
+    $providerName = App::getEnv('OPEN_RUNTIMES_PROXY_LOGGING_PROVIDER', '');
+    $providerConfig = App::getEnv('OPEN_RUNTIMES_PROXY_LOGGING_CONFIG', '');
+    $logger = null;
+
+    if (!empty($providerName) && !empty($providerConfig) && Logger::hasProvider($providerName)) {
+        $adapter = match ($providerName) {
+            'sentry' => new Sentry($providerConfig),
+            'raygun' => new Raygun($providerConfig),
+            'logown' => new LogOwl($providerConfig),
+            'appsignal' => new AppSignal($providerConfig),
+            default => throw new Exception('Provider "' . $providerName . '" not supported.')
+        };
+
+        $logger = new Logger($adapter);
+    }
+
+    return $logger;
+});
+
+$register->set('algo', function () {
+    $algoType = App::getEnv('OPEN_RUNTIMES_PROXY_ALGORITHM', '');
+    $algo = match ($algoType) {
+        'round-robin' => new RoundRobin(0),
+        'random' => new Random(),
+        default => new Random()
+    };
+
+    return $algo;
+});
+
+// Setup Resources
+App::setResource('state', fn () => $register->get('state'));
+App::setResource('logger', fn () => $register->get('logger'));
+App::setResource('algo', fn () => $register->get('algo'));
+
+// Balancing must NOT be registry. This has to run on every request
+App::setResource('balancing', function (Table $state, Algorithm $algo) {
+    $balancing = new Balancing($algo);
+
+    $balancing->addFilter(fn ($option) => $option->getState('status', 'offline') === 'online');
+
+    foreach ($state as $state) {
+        $balancing->addOption(new Option($state));
+    }
+
+    return $balancing;
+}, ['state', 'algo']);
+
+function fetchExecutorsState(Registry $register, bool $forceShowError = false): void
```

Comment:
```suggestion
function healthcheck(Registry $register, bool $forceShowError = false): void
```

Reply:
Renamed to `healthCheck`. (its not 1 word, source Traefik)